src/domain/ecoverse/ecoverse.service.ts

```diff
@@ -176,6 +128,18 @@ export class EcoverseService {
     }
   }
 
+  async getOrganisations(): Promise<IOrganisation[]> {
+    try {
+      const ecoverse: IEcoverse = await this.getEcoverse();
+
+      if (!ecoverse.organisations) throw new Error('Unreachable');
```

Comment:
Why not returning empty array instead of throwing error? 
What does 'Unreachable' mean in this context? It sounds like network error.

Reply:
Organisations are not initialised in the constructor due to restrictions from TypeORM - but are in the intialiseMembers method. So they are guaranteed to be in place.